docs/modules/traits/pages/configuration.adoc

```diff
@@ -0,0 +1,36 @@
+= Configuration Trait
```

Comment:
It seems that file should be removed?

Reply:
Yeah, the generate-doc does not clean apparently